migrations/001_create_tables.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE User (
+    ID int NOT NULL,
+    login varchar(255) NOT NULL,
+    email varchar(255) NOT NULL,
+    password varchar(255)
+);
```

Comment:
Please add new line to the end of the file 
![image](https://user-images.githubusercontent.com/17098967/185766046-8ba1a0db-907a-4d32-9b80-f42ac7c20658.png)